src/Core/Data/Math.hpp

```
@@ -14,7 +15,10 @@
 #include <Core/Utils/SystemUtils.hpp>
 
 
-#define PI 3.14159265358979323846
+inline const double EPSILON = std::numeric_limits<double>::epsilon();   // Epsilon value, as defined by std::numeric_limits
+inline const double R_EPSILON = 1e-11;                                  // "Realistic" epsilon with a safer tolerance for astronomical calculations
+inline const double PI = std::numbers::pi;
```

Clang (14.0.6)

[warning] variable name 'PI' is too short, expected at least 3 characters

(readability-identifier-length)